db/migrate/20250114210637_add_claims_api_processes.rb

```diff
@@ -0,0 +1,14 @@
+class AddClaimsApiProcesses < ActiveRecord::Migration[7.2]
+  def change
+    create_table :claims_api_processes, id: :uuid do |t|
+      t.uuid :processable_id, null: false
+      t.string :processable_type, null: false
+      t.string :step_type
+      t.string :step_status
+      t.datetime :completed_at
+      t.jsonb :error_messages, default: []
+
+      t.timestamps
+    end
```

Comment:
@tycol7 What is the reason to have `completed_at` and timestamps since timestamps includes created_at?